### PR TITLE
Add Empty GRPC Metering Report Service [DPP-856]

### PR DESCRIPTION
**Pull request by simonmaxen-da**
### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description
- [ ] If you mean to change the status of a component, please make sure you keep [the Component Status page](https://github.com/digital-asset/daml/blob/main/docs/source/support/component-statuses.rst) up to date.

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

**Review by mziolekda — APPROVED** [review]
Looks good! Great idea with the dummy report response

File: ledger/participant-integration-api/src/main/scala/platform/apiserver/services/ApiMeteringReportService.scala [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    MeteringReportServiceGrpc.bindService(this, executionContext)
+
+  override def close(): Unit = ()
+
+  override def getMeteringReport(
+      request: GetMeteringReportRequest
+  ): Future[GetMeteringReportResponse] = {
+    logger.info(s"Received metering report request: $request")
+    val generationTime =
+      com.google.protobuf.timestamp.Timestamp.of(java.time.Instant.now().getEpochSecond, 0)
```
Comment (meiersi-da) [inline ledger/participant-integration-api/src/main/scala/platform/apiserver/services/ApiMeteringReportService.scala:32]: You are using a protobuf Timestamp, which stores a nanosecond part. For uniformity of our codebase, I'd suggest to store the nanoseconds as well, given that `java.time.Instant` provides them.

**Review by meiersi-da — APPROVED** [review]
Thanks @simonmaxen-da . LGTM up to the one comment.

File: ledger/participant-integration-api/src/main/scala/platform/apiserver/services/ApiMeteringReportService.scala (hunk 1 above)
Comment (simonmaxen-da) [reply ledger/participant-integration-api/src/main/scala/platform/apiserver/services/ApiMeteringReportService.scala:32]: Done.

**Pull request merged**